plugins/engines/optix/OptiXScene.cpp

```diff
@@ -891,4 +891,9 @@ uint64_t OptiXScene::_getBvhSize( const uint64_t nbElements ) const
     return 64 * 1.3f * nbElements;
 }
 
+void OptiXScene::saveSceneToCacheFile()
+{
+    BRAYNS_ERROR << "OptiXScene::saveSceneToCacheFile not implemented" << std::endl;
```

Comment:
Yes, but it needs a different implementation for every engine. For the moment, only OSPRay support large models, and requires the use of a cache.